src/core/gui/XournalView.cpp

```diff
@@ -139,6 +139,45 @@ auto XournalView::onKeyPressEvent(GdkEventKey* event) -> bool {
         }
     }
 
+    if (auto* tool = getControl()->getWindow()->getPdfToolbox(); tool->hasSelection()) {
+        if (event->keyval == GDK_KEY_c && event->state & GDK_CONTROL_MASK) {
+            // Shortcut to get selected PDF text.
+            tool->copyTextToClipboard();
+            return true;
+        }
+    }
+
+    if (auto* selection = getSelection(); selection) {
+        if (event->keyval == GDK_KEY_Escape) {
+            clearSelection();
+            return true;
+        }
+
+        int d = 3;
+        if (event->state & GDK_MOD1_MASK) {
+            d = 1;
+        } else if (event->state & GDK_SHIFT_MASK) {
+            d = 20;
```

Comment:
Could you make those magic constants constexpr somewhere?

Reply:
Sure, I will.